sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/jdbc/JdbcPropertiesBeanPostProcessor.java

```diff
@@ -114,6 +116,8 @@ private void enhanceUserAgent(DatabaseType databaseType, JdbcConnectionStringEnh
             Map<String, String> enhancedProperties = new HashMap<>();
             enhancedProperties.put(POSTGRESQL_PROPERTY_NAME_APPLICATION_NAME,
                 AzureSpringIdentifier.AZURE_SPRING_POSTGRESQL_OAUTH);
+            enhancedProperties.put(POSTGRESQL_PROPERTY_NAME_ASSUME_MIN_SERVER_VERSION,
+                POSTGRESQL_PROPERTY_VALUE_ASSUME_MIN_SERVER_VERSION);
```

Comment:
And I think we should link the docs here.

Reply:
Agree